src/main/java/software/amazon/event/kafkaconnector/EventBridgeWriter.java

```diff
@@ -97,7 +97,7 @@ public EventBridgeWriter(EventBridgeSinkConfig config) {
     this.ebClient = client;
 
     this.eventBridgeMapper = new DefaultEventBridgeMapper(config);
-    this.batching = new SingletonEventBridgeBatching();
+    this.batching = new BestEffortInOrderEventBridgeBatching();
```

Comment:
Question: IMHO we should rename `BestEffort` to `Default...` or `Simple` because "best effort" sounds misleading. WDYT? Also, the TODO comment still needs to be udpated

Reply:
^ @maschnetwork fyi 

Reply:
It was `BestEffort` in order before I got the information of maximum item size.

+1 for `DefaultEventBridgeBatching` WDYT?

Reply:
SGTM, since we keep order across batches and use a simple algorithm to fill the batches based on EB API limits.